src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -137,36 +137,36 @@ DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwr
         THROW_IF_FAILED(textFormat->SetParagraphAlignment(DWRITE_PARAGRAPH_ALIGNMENT_NEAR));
         THROW_IF_FAILED(textFormat->SetWordWrapping(DWRITE_WORD_WRAPPING_NO_WRAP));
 
-        _textFormatMap.insert({ fontInfo, textFormat });
+        _textFormatMap.emplace(_ToMapKey(weight, style, stretch), textFormat);

```

Comment:
```suggestion
        _textFormatMap.emplace(_ToMapKey(weight, style, stretch), std::move(textFormat));
```

perhaps? this might save an AddRef and Release. Just... because.